odm_sdk/connection.py

```diff
@@ -131,6 +131,24 @@ def login_by_token(self, token):
             hostname = urlsplit(self.server_url).hostname
             raise GenestackAuthenticationException('Fail to login by token to %s' % hostname)
 
+    def login_by_access_token(self, access_token):
+        """
+        Attempt a login on the connection with the specified access token.
+        Raises an exception if the login fails.
+
+        :param access_token: OAuth access token
+        :rtype: None
+        :raises: :py:class:`~odm_sdk.GenestackAuthenticationException` if login failed
+        """
+        access_token = os.getenv(access_token, access_token)
+        self.session.headers.update({'Authorization': f'Bearer {access_token}'})
+        logged = self.application('genestack/signin').invoke('authenticateOAuthAccessToken', access_token)
+        Connection._handle_inactive_user_auth(logged)
+        if not logged['authenticated']:
+            self.session.headers.pop('Authorization')
+            hostname = urlsplit(self.server_url).hostname
+            raise GenestackAuthenticationException('Fail to login by token to %s' % hostname)
+
```

Comment:
[Optional] extract duplicated code

Reply:
I supposed we are going to remove interactive mode from user, so there wouldn't be duplicates

Reply:
created https://genestack.atlassian.net/browse/ODM-11600 for this